manifests/charts/global.yaml

```diff
@@ -361,6 +361,10 @@ global:
   # Currently, two options are supported: "third-party-jwt" and "first-party-jwt".
   jwtPolicy: "third-party-jwt"
 
+  # Configure External Ca Integration with Istiod as RA
+  ca: {}
```

Comment:
we don't want to introduce new API in values.yaml - use env vars if you need an informal API for testing.

Reply:
Noted. Will be merging the configuration changes in a seperate PR